wordpress-mobile/gutenberg-mobile
# Fix SelectControl to display label instead of value

Fixes #1764 

Related gutenberg PR: https://github.com/WordPress/gutenberg/pull/19547

## Description
This PR fixes the `SelectControl` to display the `label` of the currently selected option, instead of the `value`. Issue: https://github.com/wordpress-mobile/gutenberg-mobile/issues/1764.

## How has this been tested?
* View the block settings for Gallery or Video block
* Make sure `SelectControl` ("Preload" setting on video block, "Link To" setting on gallery block) displays title-case content.

PR submission checklist:
